src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -111,7 +117,7 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
   const [groupCreationState, setGroupCreationState] = useState<GroupCreationModalState>(
     GroupCreationModalState.DEFAULT,
   );
-  const [isCellsOptionEnabled, setIsCellsOptionEnabled] = useState(true);
+  const [isCellsOptionEnabled, setIsCellsOptionEnabled] = useState(isCellsEnabledForTeam);
```

Comment:
initialized with `false` if cells isn't enabled for the team and `true` otherwise

Reply:
Why not use `enableCellsToggle`?
```suggestion
  const [isCellsOptionEnabled, setIsCellsOptionEnabled] = useState(enableCellsToggle);
```

Reply:
should be the same in practice but that's 1 additional safety layer, I like it :+1:

Reply:
addressed here https://github.com/wireapp/wire-webapp/pull/19333/commits/6b013466ce377e55e974c04c115afd57f06fa980